### ISSUE TITLE
Insecure default permissions for `/root` in default overlays

### ISSUE THREAD

**Issue by upsj** [warewulf/warewulf#1452]
### Steps to reproduce

1. Install Warewulf
2. Check the permissions of `/var/lib/warewulf/overlays/generic/rootfs/root` (these will be used to overwrite the container-provided permissions when applying the runtime overlay). The directory should not be readable by all users.

### Error message

```
total 8
drwxr-xr-x. 2 root root 4096 Oct 13 08:31 etc
drwxr-xr-x. 3 root root 4096 Oct 13 08:31 root
```

### Information on your system

```
wwctl version:   4.5.8-1
rpc version: apiPrefix:"rc1"  apiVersion:"1"  warewulfVersion:"4.5.8-1"
```

```
NAME="Rocky Linux"
VERSION="9.4 (Blue Onyx)"
ID="rocky"
ID_LIKE="rhel centos fedora"
VERSION_ID="9.4"
PLATFORM_ID="platform:el9"
PRETTY_NAME="Rocky Linux 9.4 (Blue Onyx)"
ANSI_COLOR="0;32"
LOGO="fedora-logo-icon"
CPE_NAME="cpe:/o:rocky:rocky:9::baseos"
HOME_URL="https://rockylinux.org/"
BUG_REPORT_URL="https://bugs.rockylinux.org/"
SUPPORT_END="2032-05-31"
ROCKY_SUPPORT_PRODUCT="Rocky-Linux-9"
ROCKY_SUPPORT_PRODUCT_VERSION="9.4"
REDHAT_SUPPORT_PRODUCT="Rocky Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="9.4"
```

### General information

- [X] I have run `wwctl version` and reported the contents of `/etc/os-release`
- [X] I have searched the issues of this repo and believe this is not a duplicate
- [X] I have captured and reported relevant error messages and logs

**Comment by anderbubble** [comment]
#1454 was merged to v4.5.x, but this needs to be done in main.

**Comment by upsj** [comment]
See https://github.com/warewulf/warewulf/pull/1456
